code/iaas/agent-instance/src/main/java/io/cattle/platform/agent/instance/factory/impl/AgentInstanceBuilderImpl.java

```diff
@@ -50,7 +52,11 @@ public AgentInstanceBuilderImpl(AgentInstanceFactoryImpl factory, Instance insta
         this(factory);
         this.accountId = instance.getAccountId();
         this.zoneId = instance.getZoneId();
-        this.uri = "event:///instanceId=" + instance.getId();
+        String uriPrefix = "event";
+        if (StringUtils.equals(SystemContainer.LoadBalancerAgent.name(), instance.getSystemContainer())) {
```

Comment:
@ibuildthecloud I can use system label to pass this info if you are ok with this approach